src/stash-list/folder.vue

```diff
@@ -26,17 +26,19 @@
     <!-- This is at the end so it gets put in front of the buttons etc.
          Important to ensure the focused box-shadow gets drawn over the buttons,
          rather than behind them. -->
-    <editable-label :class="{'folder-name': true, 'ephemeral': true}"
-                    :value="nonDefaultTitle"
-                    :defaultValue="defaultTitle"
-                    :edit="rename"></editable-label>
+    <editable-label 
+        :class="{'folder-name': true, 'ephemeral': true, 'has-tabs': openTabsCount > 0 }"
```

Comment:
I think this would be easier/clearer to style (and make the styling easier to change in the future) if you call the class `has-open-tabs` and put it on the parent `<section>` element. Then the CSS can style `.has-open-tabs .folder-name` (or whatever other child elements).

Reply:
I changed it. However, there actually is no `disabled` class in editable label (unless I'm blind). It's an attribute on the tag. I changed the CSS to match against the span tag that only exists when it's not being edited.

Reply:
The `disabled` class is used here for "Unstashed Tabs" (/ "Open Tabs"). (Honestly, this should probably not even be an `<editable-label>`; I think the only reason it is is because `folder.vue` and `window.vue` used to be the same component. So this is probably cruft leftover from the split.)

https://github.com/josh-berry/tab-stash/blob/357a9f4a18279a6f00d99206249c57123b2ea8ac/src/stash-list/window.vue#L35-L36

So I think you could still get away with moving `has-tabs`/`has-open-tabs` to the parent element without worrying about `disabled`, since it would never be set at the same time as `disabled`.

Reply:
I will take another look at what you mentioned. Probably tomorrow though, and after the container color stuff is merged (as that seems imminent) since I'll have to rebase this and fix conflicts anyway.

Reply:
Was 62e1534 the change you were looking for here? The `has-open-tabs` class is already applied to the section (don't remember exactly when I made that change.)

Reply:
I think it was just moving `has-open-tabs` to the parent `<section>` (and corresponding style changes); I don't have a strong opinion on https://github.com/josh-berry/tab-stash/commit/62e15347e0c69957075424a051616d72b199f341, it seems perfectly fine to me. I'll re-review later tonight but my cursory glance suggests this one is addressed.

Reply:
Yeah, I don't think it makes a difference (at least with how stuff is set up currently.) I just did it because I thought that's what you were asking for. :)